### DISCUSSION TITLE
Integrate another product add-on plugin , to solve webview checkout issues with woocommerce product-addon plugin

### DISCUSSION THREAD

**Discussion by ahmadmaaly** [inspireui/support#5224] [New Feature/Ideas Requests]
woocommerce product-addon plugin has many issues with webview checkout , it’s not send all values to the webview checkout , and i contact support and there are no solution for it

can you integrate other plugin please , like this :

https://wordpress.org/plugins/advanced-product-fields-for-woocommerce/ https://wordpress.org/plugins/woo-extra-product-options/

**Comment by CeoAlexMartinez** [comment]
esperando esta funcion

**Comment by antimimin** [comment]
Yes, it has issue with webview checkout. But I think InspireUI Team will find the way to fix it. Because I use Cirilla too, it works in webview checkout.
